xml/planning-architecture-alternative-rhel_compute_model.xml

```diff
@@ -6,6 +6,153 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>&rhla; Compute Nodes</title>
+ <para>
+  There are several tasks you need to perform in order to install the software
+  required for &rhel; &compnode;s. Prior to this, you need to configure &obs;
+  &centos; software repository on the deployer node and on the &rhel; &compnode;s
+  using the following URL:
+  <link xlink:href="https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/Cent OS/CentOS_7.3">https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/Cent OS/CentOS_7.3</link>
+ </para>
+
+  <procedure>
+   <step>
+    <para>
+     &ostack; service and its components are installed on &rhel; &compnode;s
+     via virtualenv (venv). So you need to install the following venv packages
+     from &obs; &centos; repository on the deployer node:
+    </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-nova-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-neutron-rhel-x86_64
+      </para>
+      </listitem>
+      <listitem>
+       <para>
+	venv-openstack-monasca_agent-rhel-x86_64
+      </para>
+      </listitem>
+     </itemizedlist>
+    <para>
+     On the deployer node, &obs; &centos; repository needs to be configured with
+     lower priority than &sles; repositories. This ensures that
+     packages present in both repos are preferably installed from the &sles; repository.
+    </para>
+
+     <screen>zypper addrepo --name centos_c8_ardana --priority 9999 https://download.opensuse.org/repositories/systemsmanagement:/Ardana:/8:/CentOS/CentOS_7.3/centos_c8_ardana</screen>
```

Comment:
Please add a \\ char + line break before the long URL.

(Also, the indentation is one space off here.)